### ISSUE TITLE
Build issue with hyperscan 5.1.0-1 and gohs v1.1.1

### ISSUE THREAD

**Issue by jmdacruz** [flier/gohs#41]
I'm seeing the following error when using gohs `v1.1.1`, hyperscan `5.1.0-1` and the base `golang:1.16.6` docker image:

```
/go/pkg/mod/github.com/flier/gohs@v1.1.1/hyperscan/internal_v5.go:41:9: could not determine kind of name for C.hs_compile_lit
/go/pkg/mod/github.com/flier/gohs@v1.1.1/hyperscan/internal_v5.go:88:9: could not determine kind of name for C.hs_compile_lit_multi
```

# How to reproduce?

## Files

* `Dockerfile`
```dockerfile
FROM golang:1.16.6
RUN apt -yq update && apt -yq install libhyperscan-dev=5.1.0-1 libhyperscan5=5.1.0-1

ADD . /src
WORKDIR /src
RUN go build ./...
```

* `go.mod` (run `go mod tidy` to generate `go.sum`)
```go.mod
module example.com

go 1.16

require github.com/flier/gohs v1.1.1
```

* `main.go`
```golang
package main

import (
	"github.com/flier/gohs/hyperscan"
)

func main() {
	// simple app to trigger build with gohs and hyperscan
	hyperscan.NewLiteral("something", hyperscan.Caseless)
}

```
## Running
Do:
```shell
docker build .
```

**Comment by jmdacruz** [comment]
Doing a bit of digging into the hyperscan codebase, it looks like the APIs references used in gohs v1.1.1 where only added in 5.1.2 (e.g., `hs_compile_lit`), which means that gohs v1.1.1 can no longer support hyperscan versions <= 5.1.1

**Comment by flier** [comment]
Well, I only tested 4.7, 5.2 and 5.4, I will add a new build tag for 5.1.2 later

**Comment by flier** [comment]
Fixed in v1.2.0
